src/test/java/uk/gov/hmcts/reform/em/stitching/service/CdamServiceTest.java

```diff
@@ -105,21 +105,21 @@ public void downloadFile() throws Exception {
                 .getMetadataForDocument("xxx", "serviceAuth", docStoreUUID);
     }
 
-    @Test(expected = DocumentTaskProcessingException.class)
-    public void downloadFileNullResponseBody() throws Exception {
+    @Test()
+    void downloadFileNullResponseBody()  {
```

Comment:
Could we make the test name pertinent to what this will test? I.e. throwing a DocumentTaskProcessingException. E.g. testDownloadFileNullResponseBodyThrowsDocumentTaskProcessingException?